app/models/journal.rb

```diff
@@ -0,0 +1,3 @@
+class Journal < ApplicationRecord
+  belongs_to :user
```

Comment:
The user writes a journal entry about the client. So the user is the author, and it is about the client.